### ISSUE TITLE
Exception raised when calling destroy "NameError - instance variable @scope_changed not defined:"

### ISSUE THREAD

**Issue by naysayer** [swanandp/acts_as_list#206]
I recently upgraded from v0.6.0 to v0.7.3 and begun to notice an issue when calling destroy on an instance that was using acts_as_list. As stated in the title, the exception raised is "NameError - instance variable @scope_changed not defined:" This issue only effect instances from classes that uses acts_as_list. Reverting back to my original version of 0.6.0 solved the issue for the time being. 

Other Info:
- All classes that use acts as list use a scope to a parent class. 
- Ruby version 2.3.0
- Rails version 4.2.6

**Comment by fabn** [comment]
See #199 and [changelog](https://github.com/swanandp/acts_as_list/blob/master/CHANGELOG.md#v074-2016-04-15)

Version 0.7.3 [has been yanked](https://rubygems.org/gems/acts_as_list/versions/0.7.3)

## TLDR;

Upgrade to v0.7.4
